index.html

```diff
@@ -261,44 +261,44 @@ <h3 class="connect">
       <section class="language">
 
         <div class="language-header">
-          <li>Languages</li>
+          <h1>Languages</h1>
```

Comment:
- [ ] As suggested by the previous reviewer that you should change your `language, frameworks and skills` to an [HTML heading ](https://www.w3schools.com/html/html_headings.asp), I would kindly suggest you make use of a different `hx` instead of the `h1` tag for semantic reasons, Because in a webpage following the best practice, it is advisable to have only one `h1` for easy accessibility and readability. 👍 👍 
Kindly use `h2` - `h6`